src/constants/libraryName.js

```diff
@@ -10,8 +10,4 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-/* #if _REACTOR
-export default "https://ns.adobe.com/experience/alloy/reactor";
```

Comment:
I'm thinking that for the launch extension, we can use the "onBeforeEventSend" function to update the implementation name and version.

Reply:
That bug is unfortunate. Yes we should fix that.

I didn't think about the optics of the onBeforeEventSend callback.